README.md

```diff
@@ -152,11 +152,13 @@ Measured::Weight.unit_names_with_aliases
   * oz, ounce, ounces
```

Comment:
We added a bunch of new weights too. Check out https://github.com/Shopify/measured/blob/master/lib/measured/units/weight.rb to see what we're missing.